src/conv.jl

```diff
@@ -48,22 +48,27 @@ export conv, conv!, ∇conv_data, ∇conv_data!, ∇conv_filter, ∇conv_filter!
 """
     conv(x, w; stride = 1, pad = 0, dilation = 1, flipped = false, groups = 1)
 
-Apply convolution filter `w` to input `x`. `x` and `w` are 3d/4d/5d tensors 
-in 1d/2d/3d convolutions respectively. 
+Apply convolution filter `w` to input `x`. `x` and `w` are 3d/4d/5d tensors
+in 1d/2d/3d convolutions respectively.
 """
-function conv(x, w::AbstractArray{T, N}; stride=1, pad=0, dilation=1, flipped=false, groups = 1) where {T, N}
-    stride = expand(Val(N-2), stride)
-    pad = expand(Val(N-2), pad)
-    dilation = expand(Val(N-2), dilation)
-    cdims = DenseConvDims(x, w; stride=stride, padding=pad, dilation=dilation, flipkernel=flipped, groups = groups)
-    return conv(x, w, cdims)
+function conv(
```

Comment:
Best to use the existing function signature style across the PR

Reply:
Reverted to the existing style.